### PR TITLE
chore(reserve): comments and rewording and final migration check

### PR DESCRIPTION
**Pull request by istae**
### Checklist

- [ ] I have read the [coding guide](https://github.com/ethersphere/bee/blob/master/CODING.md).
- [ ] My change requires a documentation update, and I have done it.
- [ ] I have added tests to cover my changes.
- [ ] I have filled out the description and linked the related issues.

### CODE REVIEW

File: pkg/storer/migration/step_06.go [hunk 1]
```diff
@@ -160,5 +161,21 @@ func addStampHash(logger log.Logger, st transaction.Storage) (int64, int64, erro
...
+	}
+
+	chunkBinCnt, err := st.IndexStore().Count(&reserve.ChunkBinItem{})
+	if err != nil {
+		return 0, 0, err
+	}
+
+	if batchRadiusCnt != chunkBinCnt {
+		return 0, 0, fmt.Errorf("index counts do not match, %d vs %d", batchRadiusCnt, chunkBinCnt)
+	}
```
Comment (acha-bill) [inline pkg/storer/migration/step_06.go:176]: This should be before the migration starts.
Then at the end, we compare that `seenCnt == doneCnt == batchRadiusCnt == chunkBinCnt`

**Review by acha-bill — CHANGES_REQUESTED** [review]

**Review by acha-bill — APPROVED** [review]

**Review by martinconic — APPROVED** [review]

**Comment by ldeffenb** [comment]
So, if I've already migrated my sepolia testnet nodes without making a backup of the pre-2.2.0 database, should I `db nuke` before loading the next RC?

**Comment by istae** [comment]
> So, if I've already migrated my sepolia testnet nodes without making a backup of the pre-2.2.0 database, should I `db nuke` before loading the next RC?

We don't think that the issue is with the migration logic. So if your node completed the migration fine, then at this point, you do not need to do anything.

**Comment by ldeffenb** [comment]
> We don't think that the issue is with the migration logic. So if your node completed the migration fine, then at this point, you do not need to do anything.

Thank you.   When the team is ready, possibly after the next RC comes out correcting whatever is going on, I can load a new version of the OSM map into the sepolia testnet which should definitely stress the new chunk timestamp re-use/update logic.

**Pull request merged**